plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -291,6 +291,7 @@ public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHand
             return tableHandle.getColumns().get();
         }
         checkArgument(tableHandle.isNamedRelation(), "Cannot get columns for %s", tableHandle);
+        verify(tableHandle.getAuthorization().isEmpty(), "authorization should be empty");
```

Comment:
"Unexpected authorization is required for table: %s"?

Start the message from capital letter.